scenarios/Azlaroc/sftpgo-bf.yaml

```diff
@@ -0,0 +1,17 @@
+type: leaky
+name: Azlaroc/sftpgo-bf
+description: "Detect SFTPGo bruteforce attacks on FTP/SSH"
+filter: "evt.Meta.log_type == 'sftpgo_auth' && evt.Meta.is_failed_login == 'true'"
+groupby: evt.Meta.source_ip
+capacity: 5
+leakspeed: "2m"
```

Comment:
`leakspeed` does not mean `5 events in 2 minutes` it means leak an event every 2 minutes.

This is quite high for a `bruteforce` scenario we would typically expect maybe `30s` , and we can also drop the `capacity` down to `3` to be a little bit more restrictive on attempts.